### DISCUSSION TITLE
Possible to run both Telegram and Whatsapp at the same time?

### DISCUSSION THREAD

**Discussion by shandsy** [d99kris/nchat#258] [Q&A]
As some of my friends are normal and some are nerds, I need to access both Whatsapp and Telegram. Is it possible to add both of these to nchat? Either in a way that let's me switch between TG/Wap with a keybinding or in a way that just displays all chats in one window. Or any other way that I'm not thinking off. 

I tried searching for this but can't seem to find anything, sorry if it has been answered already.

**Comment by d99kris** [comment]
Hi - yes it is supported. See #140 for some notes. But essentially one just need to run `nchat -s` one time for each protocol to add. One can also have several Telegram (or WhatsApp) accounts (not necessarily just one of each).

**Reply by shandsy** [reply]
Oh, I tried it, but it didn't work because nchat was already running. Turning it off and running `nchat -s` solved my issue. 

Thank you for the fast reply!
